nationality-park/webscraper.js

```diff
@@ -0,0 +1,90 @@
+import cheerio from "cheerio";
+import fetch from "node-fetch";
+import fs from "fs";
+
+const parksData = [];
+
+const fetchNationalParks = async () => {
+  const nationalParksLink =
+    "https://pl.wikipedia.org/wiki/Parki_narodowe_w_Polsce";
+  const title = "Park Narodowy";
+  const response = await fetch(nationalParksLink);
+  const html = await response.text();
+  const $ = cheerio.load(html);
+
+  $(`.wikitable tbody tr td a[title*='${title}']`).each((i, e) => {
+    //park link
+    const parkLink = `https://pl.wikipedia.org${$(e).attr("href")}`;
+    //name of park
+    const nameOfPark = $(e).text();
+    //area
+    const area = $(
+      `.wikitable tbody tr:nth-child(${i + 2}) td:nth-child(5)`
+    ).text();
+    //voivodeship
+    const voivodeship = $(
+      `.wikitable tbody tr:nth-child(${i + 2}) td:nth-child(9)`
+    ).text();
+    //date
+    const date = $(
+      `.wikitable tbody tr:nth-child(${i + 2}) td:nth-child(4)`
+    ).text();
+    const dateText = `${nameOfPark} Park Narodowy powstał w roku ${date}. `;
+
+    parksData.push({
+      name: nameOfPark,
+      date: dateText,
+      area: area,
+      voivodeship: voivodeship,
+      href: parkLink,
+      symbol: "",
+      description: "",
+      title: "",
+    });
+  });
+};
+
+const fetchNationalPark = async () => {
+  const promises = parksData.map(({ href }) => {
+    const nationalParkLink = href;
```

Comment:
useless variable

otherwise if you want to change name of destructured property, see ->
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment